### PR TITLE
[#33] Closes issue 33 for statistics page

### PR DESCRIPTION
**Pull request by miguelfmp**
lastdays
lastnight
now
today
yesterday

Currently this is how it will look on the app, the charts package allows to edit it even more and other options may look like this

hip1

If requested I can change to something else and if possible.

### CODE REVIEW

**Comment by guivazcabral** [comment]
This looks ok. I have overflowing issues on iPhone XS:
Simulator Screen Shot - iPhone Xs - 2019-05-20 at 11 21 23

Also, each section (Agora, Hoje, etc) could be more spread apart, to keep it less cluttered.

PS: let me know if you need help rebasing to fix those conflicts 😉

**Comment by miguelfmp** [comment]
@guivazcabral rebased and fixed the conflicts, I'll work on fixing now the issues you said above

**Comment by miguelfmp** [comment]
@guivazcabral fixed the issues you mentioned!

File: lib/screens/utils/widget_utils.dart [hunk 1]
```diff
@@ -5,6 +5,6 @@ Color getFireColor(String fireStatus) {
     fireStatus == null
         ? 0xFF000000
         : int.parse(
-        '0xFF${fireStatus}'),
+        '0xFF$fireStatus'),
```
Comment (guivazcabral) [inline lib/screens/utils/widget_utils.dart]: 👍

**Pull request merged**